ddtrace/appsec/processor.py

```diff
@@ -210,9 +217,51 @@ def __attrs_post_init__(self):
         # we always need the response headers
         self._mark_needed(_Addresses.SERVER_RESPONSE_HEADERS_NO_COOKIES)
 
+    def update_rules(self, new_rules):
+        # type: (List[Dict[str, Any]]) -> None
+        try:
+            self._ddwaf.update_rules(new_rules)
+        except TypeError as e:
+            log.debug("Error updating RCM rules: %s", str(e))  # noqa: G200
```

Comment:
```suggestion
            log.debug("Error updating ASM rules", exc_info=True)  # noqa: G200
```

is this method specific to RCM updates? if so can we rename the method to `_update_rules_from_rcm`.